experiments/pagerank/src/main/resources/reference.conf

```diff
@@ -0,0 +1,7 @@
+gearpump {
+  serializers {
+    "org.apache.gearpump.experiments.pagerank.PageRankController$Tick" = ""
+    "org.apache.gearpump.experiments.pagerank.PageRankWorker$UpdateWeight" = ""
```

Comment:
What is serializer "" means? Needn't special handing? If so, how about set a special annotation for serialization message instead of modifying reference.conf for normal messages?